src/main.cpp

```diff
@@ -245,6 +245,9 @@ int main(int argc, char* argv[])
         app->setApplicationVersion(QString(APP_VERSION) + appBuild);
     }
 
+    int* nullPointer = nullptr;
+    *nullPointer = 42;
```

Comment:
Oh, no wonder it went :boom: you put a :bomb: here!

Reply:
Right, that is intentional. The issue is in the lack of information in stack traces 😄